dist/linux/app.desktop

```diff
@@ -1,13 +1,15 @@
 [Desktop Entry]
-Name=Ghostty
+Version=1.0
+Name=Ghostty@@NAME@@
```

Comment:
The `@@` style substitutions are kinda cursed IMO. We should follow how most GTK/GNOME apps do it and just have separate desktop/metainfo/etc files for the release build and the debug build (cf. Ptyxis). It's also a bit misleading because usually by convention these "template" files that need substitution are marked with an `.in` suffix so that readers know that some variables are left unexpanded.

Reply:
We need some kind of templating anyways. The `@@` doesn't bother me, but we should suffix this with `.in`.

Reply:
I chose the "@@" because it's "traditional". It's a long established standard basically because "@@" is unlikely to appear in normal texts. Our needs for templating are very simple. If our needs ever get more complex we can look into any of the various templating engines that are available rather than trying to roll our own more sophisticated one.

Reply:
The "templated" files have been renamed with a `.in` suffix.